src/elements/content-sidebar/activity-feed/comment/__tests__/Comment-test.js

```diff
@@ -101,6 +101,60 @@ describe('elements/content-sidebar/ActivityFeed/comment/Comment', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
+    test('should allow actions when can_delete permission is present', () => {
```

Comment:
Since `permissions` is the only variable between these tests, they can be condensed considerably using `test.each`.  Take a look at `elements/content-sidebar/versions/__tests__/VersionsItemActions-test.js` for a basic example.

Reply:
I can definitely improve this as a follow up. Merging this since it blocks work for tasks.